src/falconpy/cloud_connect_aws.py

```diff
@@ -154,8 +154,6 @@ def provision_aws_accounts(self: object, body: dict, parameters: dict = None, **
         external_id -- AWS cross-account role secret. String.
         iam_role_arn -- ARN used for cross-account role. String.
         id -- AWS account ID. String.
-        mode -- Mode for provisioning. Allowed values are `manual` or `cloudformation`.
```

Comment:
Did these get removed intentionally? They are still in the swagger definition.

Reply:
Seeing as its still in the [docs](https://falcon.crowdstrike.com/documentation/91/cloud-workloads-discovery-for-aws-apis#enable-discover-for-aws-accounts) it may be worth leaving. The call should generate a CF template for the CX to download and apply. Otherwise the provision the required permissions manually.

Reply:
Works for me.  I'll commit a reversion for that change now.